src/main/java/org/myrobotlab/service/interfaces/SearchPublisher.java

```diff
@@ -13,7 +14,7 @@ public interface SearchPublisher extends TextPublisher, LocaleProvider {
 
   SearchResults publishResults(SearchResults results);
```

Comment:
please specify which methods are published.. and add the default implementation of the attach/remove SearchListener

Reply:
good call - updated

Reply:
still is missing the publishMethods and the attachSearchListener / removeSearchListener methods.

Reply:
search replace error - fixed

Reply:
I still don't see the publishMethods / attachSearchListener / removeSearchListener methods.  Did you miss some files when you checked them in?

Reply:
you should see it now ..